react/PaymentMethod.tsx

```diff
@@ -61,7 +61,7 @@ const PaymentMethod: FunctionComponent<Props & InjectedIntlProps> = ({
   return (
     <article className="flex justify-between">
       <div className="t-body lh-solid">
-        <p className={`${handles.paymentGroup} c-on-base`}>{paymentGroupSwitch(payment, intl)}</p>
+        <p className={`${handles.paymentGroup} ${handles.paymentGroup}--${payment.group} c-on-base`}>{paymentGroupSwitch(payment, intl)}</p>
```

Comment:
A better approach would be using https://github.com/vtex/payment-flags/blob/master/react/utils.tsx#L47

Reply:
This would create classes with spaces like `paymentGroup-Boleto Bancário` that would be impossible to target this class. Also the class would change if language changed.